<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmd_chain_dialog</class>
 <widget class="QDialog" name="cmd_chain_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>307</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="qtsprat.qrc">
         <normaloff>:/images/Play1Normal.png</normaloff>:/images/Play1Normal.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_pushButton">
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset resource="qtsprat.qrc">
         <normaloff>:/images/PauseNormal.png</normaloff>:/images/PauseNormal.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="step_pushButton">
       <property name="text">
        <string>Step</string>
       </property>
       <property name="icon">
        <iconset resource="qtsprat.qrc">
         <normaloff>:/images/StepForwardNormalBlue.png</normaloff>:/images/StepForwardNormalBlue.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="status_label">
     <property name="text">
      <string>Idle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtsprat.qrc"/>
 </resources>
 <connections/>
</ui>
